backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientDataResolver.java

```diff
@@ -27,4 +29,9 @@
     public TestEvent getLastTest(Person p) {
         return _testEventService.getLastTestResultsForPatient(p);
     }
+
+    public ApiFacility getFacility(Person p) {
```

Comment:
In this case I have to separately retrieve the entire list of facilities to populate the dropdown so all I need from the patient is the facility id. I suppose I can just grab the id like your test did. Not really sure if there are performance implications for doing it one way or the other.

Reply:
There are notional performance implications,￼but for one patient nothing will matter all that much￼.